graphqlapi/src/main/java/io/stargate/graphql/schema/DdlSchemaBuilder.java

```diff
@@ -453,6 +457,41 @@ private GraphQLFieldDefinition buildCreateTable() {
         .build();
   }
 
+  private GraphQLFieldDefinition buildCreateIndex() {
+    return GraphQLFieldDefinition.newFieldDefinition()
+        .name("createIndex")
+        .argument(GraphQLArgument.newArgument().name("keyspaceName").type(Scalars.GraphQLString))
+        .argument(
+            GraphQLArgument.newArgument().name("tableName").type(nonNull(Scalars.GraphQLString)))
+        .argument(
+            GraphQLArgument.newArgument().name("columnName").type(nonNull(Scalars.GraphQLString)))
+        .argument(GraphQLArgument.newArgument().name("indexName").type(Scalars.GraphQLString))
+        .argument(
+            GraphQLArgument.newArgument().name("customIndexClass").type(Scalars.GraphQLString))
```

Comment:
I know under the hood in QueryBuilder it's `customIndexClass` but is that how we want to expose it? For example, an SAI index would be `StorageAttachedIndex` rather than a class like `org.apache.cassandra.index.sasi.SASIIndex`. 

Maybe something like 'indexType` and then add a description explaining it can be the class name or type of index?